docs/guides/grafana.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Grafana
+lang: en-US
+meta:
+  - name: keywords
+    content: >-
+      pomerium identity-access-proxy data logging graphing grafana
+      authentication authorization
+description: >-
+  This guide covers how to use Pomerium to authenticate and authorize users of Grafana.
+---
+
+# Securing Grafana with Pomerium
+
+[Grafana](https://grafana.com/) is an open-source analytics and interactive visualization web application monitoring solution. It provides many user-contributed [Dashboards](https://grafana.com/grafana/dashboards/) that make it popular for enthusiasts as well as professionals.
+
+This guide will demonstrate how to secure an instance of Grafana behind Pomerium, and provide users with a seamless login to Grafana using your Identity Provider (**IdP**).
+
+## Before You Begin
+
+This guide begins with the following steps assumed complete:
+
+- A running instance of Pomerium. If you haven't already installed Pomerium, see our articles for installation on [Docker](/docs/install/readme.md), [Kubernetes](/docs/install/helm.md), or as an all-in-one [binary](/docs/install/binary.html).
+- Administrator access to a working Grafana instance, including write access to the config file `grafani.ini`, usually found in `/etc/grafana`.
+
+This guide uses the following temporary values in steps and configuration examples, which will need to be adjusted for your setup:
+
+- `http://grafana:3000` - The path Pomerium will use to access Grafana. This example emulates a common Docker-based setup.
+- `http://grafana.local:3000` - The path to access the Grafana interface from your local computer. We will need direct access to add at least one user before Pomerium is configured.
+- `https://grafana.localhost.pomerium.io` - The path to access Grafana through Pomerium. Change this to match the domain space Pomerium is configured in.
+
+## Enable JWT Authentication in Grafana
+
+Edit `grafana.ini` to configure [JWT authentication]. Replace `auth.localhost.pomerium.io` with the value of [`authenticate_service_url`] in your Pomerium configuration:
```

Comment:
Do we want to define JWT acronym?

Reply:
We do a bit lower, on [line 47](https://github.com/pomerium/pomerium/pull/2742/files#diff-c9db3551ad9e3b330e2cc09dd55e72febbeef768ae4048045ee596c2e73c4fc3R47)